website/views/options/inventory/equipment.jade

```diff
@@ -50,8 +56,14 @@
 
       li.customize-menu(ng-if='user.preferences.costume')
         menu.pets-menu(label='{{::label}}', ng-show='gear[klass]',
-          ng-repeat='(klass,label) in {warrior:env.t("warrior"), wizard:env.t("mage"), rogue:env.t("rogue"), healer:env.t("healer"), special:env.t("special"), mystery:env.t("mystery"), armoire:env.t("armoireText")}')
-          div(ng-repeat='item in gear[klass]')
+          ng-repeat='(klass,label) in {warrior:env.t("warrior"), wizard:env.t("mage"), rogue:env.t("rogue"), healer:env.t("healer"), special:env.t("special"), mystery:env.t("mystery"), armoire:env.t("armoireText")}', ng-controller="SortableInventoryController")
+          span(class="sort-menu") {{env.t('orderBy')}}
+            span.btn-group.btn-group-xs
+              button.btn.btn-default(ng-click='setOrder("text()")') {{env.t('name')}}
```

Comment:
Does this need to use angular to set the name? Would probably be more performant to do:

button.btn.btn-default(ng-click='setOrder("text()")')=env.t('name')